/**:
  ros__parameters:
    mode: "hexa"
    max_points: 500

    topics:
      actual_rpm: "/uav/actual_rpm"
      estimated_state: "/uav/rotor_state"
      estimated_cov: "/uav/rotor_state_covariance"
